src/SFA.DAS.Reservations.Web.UnitTests/Manage/WhenCallingGetProviderManage.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Encoding;
+using SFA.DAS.Reservations.Application.Reservations.Queries.SearchReservations;
+using SFA.DAS.Reservations.Domain.Interfaces;
+using SFA.DAS.Reservations.Infrastructure.Configuration;
+using SFA.DAS.Reservations.Web.Controllers;
+using SFA.DAS.Reservations.Web.Infrastructure;
+using SFA.DAS.Reservations.Web.Models;
+using SFA.DAS.Reservations.Web.UnitTests.Customisations;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.Reservations.Web.UnitTests.Manage
+{
+    [TestFixture]
+    public class WhenCallingGetProviderManage
+    {
+        [Test, MoqAutoData]
+        public async Task Then_Gets_List_Of_Reservations_From_Search(
+            ReservationsRouteModel routeModel,
+            ManageReservationsFilterModel filterModel,
+            SearchReservationsResult searchResult,
+            [Frozen] Mock<IMediator> mockMediator,
+            ManageReservationsController controller)
+        {
+            mockMediator
+                .Setup(mediator => mediator.Send(It.IsAny<SearchReservationsQuery>(), It.IsAny<CancellationToken>()))
+                .ReturnsAsync(searchResult);
+
+            await controller.ProviderManage(routeModel, filterModel);
+
+            mockMediator.Verify(mediator =>
+                    mediator.Send(
+                        It.Is<SearchReservationsQuery>(query => query.ProviderId == routeModel.UkPrn),//todo: yeah? or providerId?? :(
+                        It.IsAny<CancellationToken>()),
+                Times.Once);
+        }
+
+        [Test, MoqAutoData]
+        public async Task Then_Returns_List_Of_Reservations_For_All_Trusted_Employer_Accounts(
```

Comment:
Being a bit picky, but not sure if this test title is correct - I think its fair to say its getting search results for that provider, the trusted employer logic is now hidden within the jobs index refresh

Reply:
wrong